exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -439,6 +397,23 @@ void createBlockWithTransactionIgnoresInPoolTransactions(TestKit testKit) {
     assertThat(inPoolTransactions).hasSize(2);
   }
 
+  @Test
+  void getTransactionPool(TestKit testKit) {
+    // Create two blocks with no transactions, so two afterCommit transactions are stored in
+    // the transaction pool
+    Block block1 = testKit.createBlock();
+    Block block2 = testKit.createBlockWithTransactions();
```

Comment:
If we use `createBlock` for second time it would create a block with the first afterCommit transaction from pool, and we want to keep it there - that's why `createBlockWithTransactions`

Reply:
That's an unclear (from the source) distinction that warrants an explanation in the source, because it is relied upon.

Reply:
Javadoc of this method contains this distinction - `In-pool transactions will be ignored.`

Added an additional comment for this line.

Reply:
Javadoc does not show the intent of the writer of this method.